content/akeneo-web-api.yaml

```diff
@@ -2026,6 +2026,142 @@ paths:
         406:
           $ref: "#/responses/406Error"
 
+  /api/rest/v1/measure-family:
```

Comment:
all other routes are at singular (like pim_api_category_list) @skeleton made a comment on my PR to change it. I think we should stay consistent, what do you think ?